pkg/virt-controller/watch/vm/vm.go

```diff
@@ -3255,6 +3257,12 @@ func (c *Controller) sync(vm *virtv1.VirtualMachine, vmi *virtv1.VirtualMachineI
 		vm = vmCopy
 	}
 
+	if vmi == nil {
```

Comment:
What should happen if vmi isn't nil?

Reply:
This code piece of code will provide feedback to the user when the VM is stopped. But when it isn't nil, the VM won't create a VMI object.